src/OpenGL/OpenTK/Math/Quaternion.cs

```diff
@@ -67,21 +68,25 @@ public Quaternion(float x, float y, float z, float w)
             : this(new Vector3(x, y, z), w)
         { }
 
+#if NET
```

Comment:
The whole file is inside an `!NET`, so these changes shouldn't be needed.

Reply:
transient changes, dropping those